nav2_bringup/README.md

```diff
@@ -2,121 +2,86 @@
 
 The `nav2_bringup` package is an example bringup system for navigation2 applications.
 
-Notes: (December 2018, Crystal Release)
-* We recommend doing this on a Ubuntu 18.04 installation. We’re currently having build issues on 16.04. (see https://github.com/ros-planning/navigation2/issues/353)
+Notes: (June 2019, Dashing Release)
+* We recommend doing this on a Ubuntu 18.04 installation. We have build issues on 16.04. (see https://github.com/ros-planning/navigation2/issues/353)
 * This stack and ROS2 are still in heavy development and there are some bugs and stability issues being worked on, so please do not try this on a robot without taking *heavy* safety precautions. THE ROBOT MAY CRASH!
 * It is recommended to start with simulation using Gazebo before proceeding to run on a physical robot
 
 Install and build our code by following this guide:
 https://github.com/ros-planning/navigation2/blob/master/doc/BUILD.md
 
-## Launch Navigation2 in simulation with Gazebo (first time users)
+## Launch Navigation2 in simulation with Gazebo
 ### Pre-requisites:
 * Gazebo installed on the system
-* gazebo_ros_pkgs for ROS2 installed on the system
+* gazebo\_ros\_pkgs for ROS2 installed on the system
```

Comment:
escaping the `_` characters doesn't seem to be necessary.

Reply:
When I 'preview' it in my Eclipse IDE, it only looked correct with the escapes. I can get rid of them if Github displays it correctly. I'll take a look.

Reply:
fixed